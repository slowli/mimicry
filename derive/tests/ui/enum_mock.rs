use mimicry_derive::CallReal;

#[derive(CallReal)]
enum MyMock {
    Some(u32),
    None,
}

fn main() {}
