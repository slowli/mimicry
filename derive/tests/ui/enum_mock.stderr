error: can only derive `CallReal` for structs
 --> tests/ui/enum_mock.rs:4:1
  |
4 | enum MyMock {
  | ^^^^
